demo/chaincode/fpc/evaluate.cpp

```diff
@@ -215,7 +215,7 @@ void ClockAuction::DynamicAuctionState::processRegularRoundBids(
             // compute price point
             double minPrice = postedPrice_[clockRound_ - 1][eBid.territoryIndex];
             double clockPrice = clockPrice_[clockRound_][eBid.territoryIndex];
-            double pp = (eBid.demand.price_ - minPrice) / (clockPrice - minPrice);
+            double pp = (eBid.demand.price_ - minPrice) / (clockPrice - minPrice) * (100.0);
             eBid.pricePoint = (uint32_t)pp;
```

Comment:
I have to admit, i don't like type casts as they can bury easily problems: std::lround() might be nicer?

Reply:
possibly, let me change this...

Reply:
done